bakery_lint/fonttests/test_ttf.py

```diff
@@ -211,14 +213,14 @@ def containsSubstr(self, nameRecord, substr):
 
     @autofix('bakery_cli.fixers.RemoveItemsWithPlatformID1')
     def test_platformID_1_exists(self):
-        """ Check if NAME table does not contain items with platformID = 1 """
+        """ Is there any item with platformID=1 on the NAME table? """
         font = ttLib.TTFont(self.operator.path)
         items = [item for item in font['name'].names if item.platformID == 1]
-        self.assertTrue(bool(items), "`name` table records should not contain platformID='1' records")
+        self.assertTrue(bool(items), "NAME table should not contain platformID='1' records")
 
     @autofix('bakery_cli.fixers.RemoveNameRecordWithOpyright')
     def test_name_id_copyright(self):
-        """ Is there `opyright` substring nameID in nameId (10) ? """
+        """ Is there an `opyright` substring declared in name ID 10? """
```

Comment:
The use of `*opyright` is a simple hack for the regex, the message should be

`Name ID 10 incorrectly contains a copyright notice? (A common Fontlab 5x bug)`